ledger-flymake.el

```
@@ -66,8 +67,9 @@ Flymake calls this with REPORT-FN as needed."
     (kill-process ledger--flymake-proc))
   ;; Save the current buffer, the narrowing restriction, remove any
   ;; narrowing restriction.
-  (let ((source (current-buffer))
-        (file (buffer-file-name)))
+  (let* ((source (current-buffer))
+         (master (ledger-master-file))
+         (file (if (file-exists-p master) master (buffer-file-name))))
```

I wonder if it would be better to raise an error if `ledger-master-file` is set but the file is missing. Silently working around it is risky.

---

it isn't really working around, but rather falling back to the previous behaviour. another solution would be to offer the user a y-or-n prompt

---

Yes, I understand it's a fallback. However, I think _if_ it makes sense to ever use `ledger-master-file` for flymake (which seeme likely to me), then if the user has set `ledger-master-file` but not created that file, it would be better IMO if their syntax checks fail loudly due to that missing file than to succeed silently with possibly-inconsistent results.

---

so you think it would be better to error out instead of providing the user with a prompt? i'm ok with both, btw